about.html

```diff
@@ -0,0 +1,140 @@
+            </li>
+
+            <li class="links">
+              <a href="about.html">About</a>
+            </li>
+            <li class="links">
+              <a href="#">Program</a>
+            </li>
+            <li class="links">
+              <a href="#">Join</a>
+            </li>
+            <li class="links">
+              <a href="#">Sponsor</a>
+            </li>
+            <li class="links">
+              <a href="#">News</a>
+            </li>
+          </ul>
+          <div class="company">
+            <a href="index.html">Back to Home Page</a>
+          </div>
+        </nav>
+    </div>
+  </header>
+  <section class="first-section">
+    <div class="first-section-container">
+      <h1 class="open-heading">The Burning Circle Flight Academy</h1>
+      <div class="first-section-para">
+        <p>
+          The Burining Circle Flight Academy airshow is one of Zambia's first airshows 
+          to be hosted by a privately owned aviation institution. The whole essence of 
+          this event is to connect aerospace industry proffessionals across Africa and
+          the world at Large. This is also a great opportunity for youths interested in
+          aviation and becoming Pilots to learn from the best Pilots and know what it takes
+          to become a world class pilot. 
+        </p>
+      </div>
+      <div class="event-details">
+        <p>Please contact us per email for any further questions.
+        
+        </p>
+        <p class="email-address">
+          burningcircleflightacademy@gmail.com
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section>
+    <div class="flex-content">
+      <h2> The Burning Circle Flight Academy Airshow</h2>
+      <hr>
+      <p class="mobile-para">This will be the very first Airshow event to be hosted by Burning circle Flight academy the event will run from November 14th - 18th, so your full particpation and support will be appreciated.</p>
+      <p class='desktop-para'> This will be the very first Airshow event to be hosted by Burning circle Flight academy the event will run from November 14th - 18th, so your full particpation and support will be appreciated.</p>
+    </div>
+    <hr class="horizontal-line">
+      
+  </section>
+
+  <section class="past-section">
+    <div class="past-section-container flex-past">
+      <h2>Past Airshow events</h2>
+      <hr>
+      <p>Take a look at the past airshows we attended and what inspired us to host our own Airshow event as a Pilot training institution.</p>
+      <div class="pics">
+      <img src="images/airshow1.jpeg" alt="Dubai airshow2019">
+      <img src="images/airshow2.jpeg" alt="Dubai airshow2020">
+    </div>
+    </div>
+  </section>
+
+  <footer>
+    <div class="footer-flex">
+    <div class="footer-logo">
+      <a href="#"> BCFA</a>
+    </div>
+    <p class="copy-right">
+      Burning Circle Flight Academy <br> Some Rights Reserved
+    </p>
+  </div>
+  </footer>
```

Comment:
- [ ] The `Partner` section is missing from the `About` page, please update it.
- [ ] Kindly, make sure that the `footer` element has white background on `mobile` screens and a dark background on `desktop` screens (only the `about` has dark background on desktop).
- [ ] The `footer` element should be the same for the `Home` and about page, please update it.
Home page
![image](https://user-images.githubusercontent.com/33767951/141777298-2941be39-af77-481f-8748-434f228e5537.png)
About page
![image](https://user-images.githubusercontent.com/33767951/141777349-832e1986-d6a5-469f-a11f-0b387def39e6.png)

Reply:
the template doesnt have the partners section in the about section i think please have a proper look

Reply:
I feel you are being a little unfair you know i have done all thats required for me to do in this project

Reply:
My apologies if something seems a bit unclear, kindly double check the given design.
![image](https://user-images.githubusercontent.com/33767951/141791588-e523c87f-d5d2-4f78-8b0b-e408cfa1e05a.png)

Reply:
done